### PR TITLE
LRQA-76408 | Investigate failure for PageAudit#UncheckedSystemSettingsVirtualInstanceThenVisibleAtSiteScope

### PR DESCRIPTION
**Pull request by rafaellajordao**
Hi! :)
This pr is corresponding to this ticket: [LRQA-76408](https://issues.liferay.com/browse/LRQA-76408)

### CODE REVIEW

**Comment by rafaellajordao** [comment]
ci:test:sf

**Comment by rafaellajordao** [comment]
ci:test:tango

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/pageaudit/PageAudit.testcase [hunk 1]
```diff
@@ -4386,7 +4386,7 @@ definition {
 		}
 
 		task ("Check if Google PageSpeed Insights tab isn't visible in the 'VIRTUAL INSTANCE SCOPE' section") {
-			AssertElementPresent(
+			AssertElementNotPresent(
```
Comment (Luizcarlosqueiroz) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/pageaudit/PageAudit.testcase:4389]: When we perform the refactor of this testcase, we can remove this assert (that isn't at the end of the test only).

**Review by Luizcarlosqueiroz — APPROVED** [review]

**Comment by rafaellajordao** [comment]
> ###  ci:test:tango - 145 out of 183 jobs passed in 2 hours 30 minutes
> Click here for more details.

these failures have nothing to do with the test correction :)

**Comment by rafaellajordao** [comment]
ci:forward

**Comment by rafaellajordao** [comment]
ci:forward

**Pull request closed without merging**